lib/awful/screen.lua

```diff
@@ -483,6 +483,8 @@ end
 -- * maximized clients
 -- * floating clients
 --
+-- @DOC_screen_tiled_clients_EXAMPLE@
```

Comment:
shouldn't this example go to documentation of `awful.layout.suit.tile.right`?

Reply:
In the short term yes, in the long term, I still plan to merge https://elv13.github.io/client_tiling/dynamite.html for v5.0. It has a different doc template because these doc examples have a lot more `assert` and tests built-in.

Reply:
It definitely could. I'd like to work on the client layout documentation. We deeply need a user documentation on this API and better examples for available client layouts.

I'm not sure it would be worth the time spending in it, tho, given the fact @Elv13 will eventually merge his module... So this API will be deprecated/removed "soon".

Reply:
> "soon"

For undefined version of "soon". It can definitely be used in the meantime. You will hit some problems in the shims where some layouts are just not rendered properly without serious effort. I think there is lower hanging fruits to spend time on. The inheritance stuff is a great example of how to add value to everything with a one-time effort.